homeassistant/components/camera/skybell.py

```diff
@@ -19,26 +24,48 @@
 
 SCAN_INTERVAL = timedelta(seconds=90)
 
+IMAGE_AVATAR = 'avatar'
+IMAGE_ACTIVITY = 'activity'
+
+CONF_ACTIVITY_NAME = 'activity_name'
+CONF_AVATAR_NAME = 'avatar_name'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=[IMAGE_AVATAR]):
+        vol.All(cv.ensure_list, [vol.In([IMAGE_AVATAR, IMAGE_ACTIVITY])]),
+    vol.Optional(CONF_ACTIVITY_NAME, default=""): cv.string,
```

Comment:
Remove default, since we're not using it.

Reply:
It is used, if the name is left out of config, no name should be appended when we check if it's non empty.

Reply:
Nevermind, I see what you mean about using None instead of empty string below.